lib/rules/no-octal.js

```diff
@@ -11,6 +11,8 @@
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Octal literals are forbidden in some of the language versions, right? (Seems this should be a "suggestion" at least.)

Reply:
In strict mode (and modules) they are forbidden.

I agree, seems like suggestion is the right type.